internal/plugins/helm/v1/chartutil/chart_test.go

```diff
@@ -78,90 +71,60 @@ func TestCreateChart(t *testing.T) {
 			expectErr:        true,
 		},
 		{
-			name:               "from scaffold with apiVersion and kind",
-			group:              customGroup,
-			version:            customVersion,
+			name:               "from scaffold with kind",
 			kind:               customKind,
-			expectResource:     mustNewResource(customGroup, customVersion, customKind),
 			expectChartName:    customExpectName,
 			expectChartVersion: "0.1.0",
 		},
 		{
 			name:               "from directory",
 			helmChart:          filepath.Join(".", "testdata", chartName),
-			expectResource:     mustNewResource(chartutil.DefaultGroup, chartutil.DefaultVersion, expectDerivedKind),
 			expectChartName:    chartName,
 			expectChartVersion: latestVersion,
 		},
 		{
 			name:               "from archive",
 			helmChart:          filepath.Join(".", "testdata", fmt.Sprintf("%s-%s.tgz", chartName, latestVersion)),
-			expectResource:     mustNewResource(chartutil.DefaultGroup, chartutil.DefaultVersion, expectDerivedKind),
 			expectChartName:    chartName,
 			expectChartVersion: latestVersion,
 		},
 		{
 			name:               "from url",
 			helmChart:          fmt.Sprintf("%s/%s-%s.tgz", srv.URL(), chartName, latestVersion),
-			expectResource:     mustNewResource(chartutil.DefaultGroup, chartutil.DefaultVersion, expectDerivedKind),
 			expectChartName:    chartName,
 			expectChartVersion: latestVersion,
 		},
 		{
 			name:               "from repo and name implicit latest",
 			helmChart:          "test/" + chartName,
-			expectResource:     mustNewResource(chartutil.DefaultGroup, chartutil.DefaultVersion, expectDerivedKind),
-			expectChartName:    chartName,
-			expectChartVersion: latestVersion,
-		},
-		{
-			name:               "from repo and name implicit latest with apiVersion",
-			helmChart:          "test/" + chartName,
-			group:              customGroup,
-			version:            customVersion,
-			expectResource:     mustNewResource(customGroup, customVersion, expectDerivedKind),
```

Comment:
How are we ensuring that the tool still able to create the GKV based on the char info?

Reply:
Do you mean the default values in case only a chart was provided?

https://github.com/operator-framework/operator-sdk/blob/741d436c51126dcdfb7a5529beaeaceab9855f9f/internal/plugins/helm/v1/api.go#L141-L181

Reply:
There is no testdata verifying this, and I think that is the reason for #4575. This should probably be addressed as a separate PR, either before this gets merged or as a follow up.